tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -765,7 +765,7 @@ std::unique_ptr<ROOT::Experimental::RNTupleModel> ROOT::Experimental::RNTupleDes
    auto model = std::make_unique<RNTupleModel>();
    for (const auto &topDesc : GetTopLevelFields()) {
       auto field = Detail::RFieldBase::Create(topDesc.GetFieldName(), topDesc.GetTypeName());
-      model->AddField(std::move(field));
+      model->AddField(field.Unwrap());
```

Comment:
What is the behavior in case Create 'fails'?

Reply:
An exception is thrown in `Unwrap()` (previously `R__ASSERT` was called in `Detail::RFieldBase::Create`).